Sloth.Core/Services/CyberSourceBankReconcileService.cs

```diff
@@ -218,6 +222,21 @@ private async Task ProcessReport(Report report, Integration integration, ILogger
                     tran.Rollback();
                 }
             }
+
+            // save copy to blob storage
```

Comment:
Wrap in a try catch, log any error and continue?

Reply:
> Wrap in a try catch, log any error and continue?

Sounds good. I originally had it in a finally block but decided that was unnecessary, since `Exception` is caught just above and not rethrown.

> The only thing I might change is moving the ReportBlobContainer setting out of the CybersourceOptions as they are really a storage option .

It's just the same as ScrubberBlobContainer in the KfsScrubberOptions. Not sure if it's worth moving both of those into separate options classes that end up getting passed into the same services.